main.go

```diff
@@ -3,34 +3,22 @@ package main
 import (
 	"fmt"
 	"net/http"
-	"time"
 
-	log "github.com/Sirupsen/logrus"
-	http_router "github.com/julienschmidt/httprouter"
+	"github.com/Sirupsen/logrus"
+	"github.com/julienschmidt/httprouter"
+
+	"./config"
```

Comment:
Local imports are not recommended. Since there could be some particular changes in your local file on the disk that is making the go project compile without error.
It is always recommended to use the full import path like github.com/ashwinspg/explore-golang.git/config

Reply:
I'll work on this improvement...

Reply:
you can see the changes [here](https://github.com/ashwinspg/explore-golang/pull/2/commits/403aed9c8b270bc75a7adcda122f55ac51f71bb7)